Platform/TigerlakeBoardPkg/AcpiTables/SsdtRtd3/Rtd3I2cTouchDev.asl

```diff
@@ -0,0 +1,74 @@
+/** @file
+  ACPI RTD3 I2C Touch Device SSDT table

```

Comment:
I think it is not necessary to add this file for S0ix support.

Reply:
Done removed it, thanks.